contracts/evoting/controller/handlers.go

```diff
@@ -80,21 +80,23 @@ func (h *votingProxy) CreateElection(w http.ResponseWriter, r *http.Request) {
 // the DKG actor.
 // Body: hex-encoded electionID
 func (h *votingProxy) OpenElection(w http.ResponseWriter, r *http.Request) {
-	electionIDHex, err := ioutil.ReadAll(r.Body)
+	electionIDBuf, err := ioutil.ReadAll(r.Body)
```

Comment:
ReadAll returns a byte array, not a hex-string. I adopted the convention of writing `Buf` for byte arrays but maybe that's what `Hex` meant also?

Reply:
oh I see why you did that. Here we have a buff because that's how it is encoded in the body, but this buff represents an hex-encoded string:

```go
buff, err := ioutil.ReadAll(r.Body)
electionIDHex := string(buff)
```

Reply:
Ah I see. Since we always use `electionID` for the hex-encoded string by convention, this `electionIDHex` threw me off a bit!